client/web/app/index.html

```diff
@@ -109,6 +109,8 @@ <h3 >Może obejrzymy prezentację?</h3>
 
   </template>
   <script src="scripts/app.js"></script>
+  <script src="scripts/pdf.js"></script>
+  <script src="scripts/pdf.worker.js"></script>
 </body>
 
 </html>
```

Comment:
te dwa pliki z pdf.js to można jeszcze z bower'a ciagnac. Minus jest taki ze nie jest too oryginalny projekt, tylko taki właśnie build bowerowy: https://github.com/joseym/pdfjs-bower